### ISSUE TITLE
Unable to log in on iOS 17

### ISSUE THREAD

**Issue by MakeevaM** [mattermost/mattermost-mobile#7612]
#### Summary
Unable to log in on iOS 17

### Environment Information
  - Device Name: iPhone 13 mini
  - OS Version: 17.0
  - Mattermost App Version: 2.9.0 (build 489) from AppStore
  - Mattermost Server Version: no information available

#### Steps to reproduce
1) Open Mattermost
2) Specify the server
3) Specify the server name
4) Log in to the server
5) Close the Mattermost application (unload from memory)
6) Open again

#### Expected behavior
Successful work in authorized mode, viewing the history of channels and private dialogues

#### Observed behavior (that appears unintentional)
A window opens for entering the server address and name. When specifying the previously used ones, an error is displayed that I am already connected to the server. There is no way to exit this screen, view channels or private dialogs

**Comment by enahum** [comment]
Did you transfer phones? Uninstall and install again

**Comment by MakeevaM** [comment]
The phone was rebooted. Reinstalling is performed every time authorization is attempted and it does not help. After authorization, if you do NOT download the application, then only receiving messages is available for a while, you cannot send messages from phone

**Comment by MakeevaM** [comment]
https://github.com/mattermost/mattermost-mobile/assets/80320181/3d8e476f-a51e-48aa-b564-92e59f8fa2b2

**Comment by MakeevaM** [comment]
[com.mattermost.rn 2023-10-18--07-48-51-774.log](https://github.com/mattermost/mattermost-mobile/files/13003872/com.mattermost.rn.2023-10-18--07-48-51-774.log)

**Comment by amyblais** [comment]
Opened https://mattermost.atlassian.net/browse/MM-55011.

**Comment by enahum** [comment]
> Opened https://mattermost.atlassian.net/browse/MM-55011.

@amyblais this issue does not need a ticket, it looks to be very specific to this user/device

**Comment by MakeevaM** [comment]
Hello everyone
Is there a way around this problem? Logging in to this server is critical for me.
Reconnecting to another network (using a mobile network instead of wifi), turning on/off the VPN client, rebooting the device, repeatedly reinstalling Mattermost did not work

**Comment by enahum** [comment]
Based on what I've seen in your logs, the app is having issues creating the network client to connect to the server but it does not dtate the reason why, so I wonder if there is a setting on your device preventing this from happening.

Im running the app on iOS 17.0.3 no issues

**Comment by MakeevaM** [comment]
No, there are no specific settings, I use only the server host, the name, and I log in to gitlab. The user account is correct, I am currently logged in to gitlab under it
